frappe/desk/form/load.py

```diff
@@ -50,7 +50,6 @@ def getdoc(doctype, name, user=None):
 		get_docinfo(doc)
 
 	except Exception:
-		frappe.errprint(frappe.utils.get_traceback())
 		raise
```

Comment:
Think we could've gotten rid of this `try-except` block too.

Reply:
ooh, missed this one 😅

Reply:
will commit to `develop` directly.

Reply:
Done https://github.com/frappe/frappe/commit/06ce1f33ec0aac7b6d6281911fd70e99999120b7